package.json

```diff
@@ -54,12 +54,12 @@
 		"regenerator-runtime": "^0.13.5",
 		"splitpanes": "^2.3.6",
 		"string-length": "^5.0.0",
-		"striptags": "^3.1.1",
+		"striptags": "^3.2.0",
```

Comment:
well, `npm update` was maybe a tad too much. but let's see if this gives us a working build